src/components/Countdown.tsx

```diff
@@ -5,7 +5,7 @@ const Countdown = styled.div`
     font-weight: 500;
 `;
 const CountdownTimer: React.FC = () => {
-    const [countdownDate, setCountdownDate] = useState(new Date("Apr 6, 2024 00:00:00").getTime());
+    const [countdownDate, setCountdownDate] = useState(new Date("Apr 11, 2025 00:00:00").getTime());
     const [secondDate] = useState(new Date("Apr 7, 2024 15:37:25").getTime());
```

Comment:
Tutaj warto podmienić na 12, tak jak to było wcześniej. countdownDate -> 11, secoundDate ->12. Zgaduję, że timer jest  do podmiany kiedy rozpocznie się cały event

Reply:
znaczy wczesniej bylo wlasnie na 6, a event byl 6-7 wiec nwm szczerze to jak uwazasz dac 12?

Reply:
Chodzi o drugi parametr, secoundDate